.github/workflows/terraform.yml

```diff
@@ -0,0 +1,72 @@
+name: 'Terraform'
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    paths:
+    - 'config/terraform/aws/*'
+
+defaults:
+  run:
+    shell: bash
+    working-directory: config/terraform/aws
+
+env:
+  AWS_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
```

Comment:
Will these still populate for PRs?

Reply:
Yes, as long the compare branch isn't from a fork.